inception-project/inception
# #2689 - New export page does not work over HTTPS

**What's in the PR**
- Use wss for HTTPS

**How to test manually**
* Deploy INCEpTION behind a reverse proxy ands see that e.g. the export page works

**Automatic testing**

**Documentation**
